chapters/annotations.tex

```diff
@@ -1476,16 +1476,20 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
 
 A simple type or component of a simple type may have:
 \begin{lstlisting}[language=modelica]
-/*literal*/ constant Boolean absoluteValue = true;
+/*literal*/ constant Boolean absoluteValue;
 \end{lstlisting}%
 \annotationindex{absoluteValue}
 
-If \lstinline!false!, then the variable defines a relative quantity, and if \lstinline!true! an absolute quantity.
+If \lstinline!false!, then the component defines a relative quantity, and if \lstinline!true! an absolute quantity.
+By default, the \lstinline!absoluteValue! status is inferred by the tool.
+
+When converting between units (e.g., in plots and where parameters are edited), the unit offset must be ignored for relative quantities.
+If the \lstinline!absoluteValue! of a component is neither determined by annotation nor inference, unit conversions that would differ depending on \lstinline!absoluteValue! cannot be performed.
```

Comment:
```suggestion
For a component where unit conversions involving non-zero offset is possible (mainly temperatures) it is recommend to give a value for absoluteValue in annotation either for the component or the type declaration. Tools may infer absoluteValue in other cases, but specifying absoluteValue reduces impact of quality-of-implementation in tool ability to infer absoluteValue.
```

Reply:
Maybe we can discuss this at the monthly meeting tomorrow?

Reply:
Language group:
- Don't do temperature-conversion if absoluteValue is unknown
- Should it be possible to check/infer absoluteValue? Yes, details to be combined with unit-checking

Reply:
I have now toned down the inference part, while also being more clear about inheritance.  I have tried to do this without letting too much of the non-normative content spill out to the normative text.

Could we please resolve this conversation, and start a new one if needed, based on the new formulation with down-toned inference?

Reply:
Yes, however, I have one minor wording issue - will open new issue for that.